### DISCUSSION TITLE
Is it possible to use tickdata with pybroker?

### DISCUSSION THREAD

**Discussion by TwoPrismsOneGround** [edtechre/pybroker#17] [General]
Going through the documentation for data sources here: 

https://www.pybroker.com/en/latest/reference/pybroker.data.html#pybroker.data.DataSource

It seems that the lowest resolution is down to the second. Is it possible to use tick data with pybroker?

**Comment by edtechre** [comment]
Hi @TwoPrismsOneGround,

The seconds resolution is only used when extending the DataSource interface. However, you can use a Pandas DataFrame as a DataSource, which allows you to use any time resolution. Please refer to the notebook on [Creating Custom DataSources](https://www.pybroker.com/en/latest/notebooks/7.%20Creating%20a%20Custom%20Data%20Source.html#Using-a-Pandas-DataFrame) for more information. Keep in mind that you will still need to aggregate your tick data into OHLCV bars.
